### PR TITLE
Feat/aut 4631 4638 4641 integration

### PR DESCRIPTION
**Pull request by Karol-Stelmaczonek**
Ticket: 
https://oat-sa.atlassian.net/browse/AUT-4631
https://oat-sa.atlassian.net/browse/AUT-4641
https://oat-sa.atlassian.net/browse/AUT-4638

## What's Changed

- Integration branch

## Related PRs
- https://github.com/oat-sa/extension-tao-itemqti/pull/3052
- https://github.com/oat-sa/extension-tao-itemqti/pull/3056
- https://github.com/oat-sa/extension-tao-itemqti/pull/3058

## Summary by CodeRabbit

- **New Features**
  - Enhanced QTI Match interaction authoring with display mode (tabular/non-tabular), choice position controls, min/max associations, and an optional shuffle toggle (when available).
- **Bug Fixes**
  - Improved prompt/flow full-width layout and item ordering for associate, match, and gap-match interactions, including RTL behavior.
  - Refined Match interaction UI normalization for imported data missing mode/position styling.
  - Updated graphic gap match overlay positioning and improved spacing in match interaction property bars.
- **Tests**
  - Expanded QUnit coverage for Match and Gap Match, including editor-ready timing and prompt width/content edge cases.
- **Chores**
  - Regenerated bundled/minified CSS assets.

### CODE REVIEW

File: views/css/item-creator.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
...-indicator{width:28px;line-height:28px;height:28px;color:#919191;-moz-border-radius:2px;-webkit-border-radius:2px;border-radius:2px}.item-editor-sidebar #item-editor-item-resizer .reset-group{display:none}.item-editor-sidebar .cbr-right{width:100%}.item-editor-sidebar .cbr-right span.icon-checkbox,.item-editor-sidebar .cbr-right span.icon-radio{float:right}.item-editor-sidebar .spinner{min-width:100px;line-height:22px}.item-editor-sidebar .spinner~.tooltipstered{top:6px}.item-editor-sidebar .spinner.short{padding-right:10px}.item-editor-sidebar .min-max-panel .spinner{min-width:90px}.item-editor-sidebar .incrementer.large{min-width:55px !important}.item-editor-sidebar .incrementer-ctrl{top:-3px}.item-editor-sidebar #item-editor-font-size-changer{padding-top:1px;overflow:hidden;display:table;width:140px;float:left}.item-editor-sidebar #item-editor-font-size-changer a{display:table-cell;padding:0 10px;color:#222;text-decoration:none !important;width:70px;line-height:21px;text-align:left}.item-editor-sidebar #item-editor-font-size-changer a.icon-smaller{text-align:right}.item-editor-sidebar #item-editor-font-size-changer .icon-smaller{font-size:12px;font-size:1.2rem}.item-editor-sideb
```
[the remark this answers is not in the document]
Comment (Karol-Stelmaczonek) [reply views/css/item-creator.css]: Its bundle file which is autogenerated, skip it.

File: views/scss/qti/_graphic-gap.scss [hunk 2]
```diff
@@ -43,8 +43,8 @@
     &.qti-choices-top {
         ul.source {
             position: absolute;
-            top: -78px;
-            left: 0;
+            top: -137px;
+            left: -134px;
```
[the remark this answers is not in the document]
Comment (Karol-Stelmaczonek) [reply views/scss/qti/_graphic-gap.scss]: I will do it later.

**Review by pnal — APPROVED** [review]

**Review by tikhanovichA — APPROVED** [review]
